templates/shared/_credentials-exit-survey.html

```diff
@@ -0,0 +1,1004 @@
+<section class="p-strip">
+  <div class="row">
+    <div class="col-8">
+      <h1>{{h1}}</h1>
+      <p class="p-heading--4">{{intro_text}}</p>
+    </div>
+  </div>
+  <div class="row">
+    <div class="col-12">
+      <form action="/credentials/exit-survey" method="post">
+        <fieldset class="u-no-margin--bottom">
+          <h3>Content</h3>
+          <h4>Short Form Questions</h4>
+          <ul class="p-list">
+            <li class="p-list__item">
+              <label for="ExitSurveyRelevanceofShortFormQuestions" id="LblExitSurveyRelevanceofShortFormQuestions">This
+                exam is a
+                prerequisite for more advanced levels of certification and certifies a fundamental level of knowledge in
+                the Linux/OSS community, Ubuntu Desktop, and Ubuntu Server. How relevant are the <strong>Short Form
+                  Questions</strong> in relation to this goal. </label>
```

Comment:
Replace period with question mark.

Reply:
Replace "in the Linux/OSS community, Ubuntu Desktop, and Ubuntu Server" with "in vendor-neutral Linux as specified in our syllabus."